### PR TITLE
[Docs] improved description for fs.total.available_in_bytes

### PR DESCRIPTION
**Pull request by geekpete**
Working towards more useful descriptions for our stats as part of https://github.com/elastic/elasticsearch/issues/14576

### CODE REVIEW

File: docs/reference/cluster/nodes-stats.asciidoc [hunk 1]
```diff
@@ -96,7 +96,9 @@ information that concern the file system:
   Total number of unallocated bytes in all file stores
 
 `fs.total.available_in_bytes`::
-  Total number of bytes available to this Java virtual machine on all file stores
+  Total number of bytes available to this Java virtual machine on all file stores.
+  Depending on OS or process level restrictions, this might appear less than fs.total.free_in_bytes.
```
Comment (dakrone) [inline docs/reference/cluster/nodes-stats.asciidoc]: Can you put a backtick around `fs.total.free_in_bytes` so it shows as `<pre>`?

**Review by dakrone — APPROVED** [review]
LGTM, left one comment, feel free to merge

File: docs/reference/cluster/nodes-stats.asciidoc (hunk 1 above)
Comment (geekpete) [reply docs/reference/cluster/nodes-stats.asciidoc]: sorry, will fix this.

**Comment by geekpete** [comment]
@elasticmachine test this please

**Pull request merged**